index.rst

```diff
@@ -970,15 +1036,15 @@ Monthly Progress Narratives
 
 Every calendar month, each technical manager is required to support the
 Project Manager with a report on the activities of their group. This report
-should be generally submitted no later than the second week of the month
-(refer to the :ref:`sec-monthly-cycle`), but this may be moved earlier on
-occasion. You are encouraged to submit your report as early in the month as
+should be submitted by the 10th of the month
+(refer to the :ref:`sec-monthly-cycle`). You are encouraged to submit your report as early in the month as
 possible.
 
 Submit your report by editing the `template for the appropriate month`_ on
 Google Docs. You need to fill in all the sections with your name attached;
 when complete, remove your name. Provide a per-WBS breakdown of work over the
-month being reported on and plans for the upcoming month, as well as
+month being reported on and plans for the upcoming month, brief (1-2 sentences)
+high level summary for the Overall Summary, as well as
```

Comment:
I added such note to the template on 5/23. Plus, it was agreed at the f2f dmlt, so people should know. And yes, I will mention it when reminding people to provide the input.